core/server/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -322,6 +323,9 @@ public void removeBlocks(List<Long> blockIds) {
             }
           }
           mLostBlocks.remove(blockId);
```

Comment:
I have a question about this. Why doesn't this remove the block from being considered as lost? Shouldn't this line take care of that?